src/data/drinkRecipes/autumn.ts

```diff
@@ -0,0 +1,78 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.liquor.com/thmb/gEsRZdHlTm7j2rVHC-_QkyxZUSw=/720x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/autumn-720x720-primary-357cede4ffa9487f93136f76b21d8591.jpg"
+
+const autumn: DrinkRecipe = {
+  name: "Autumn",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Applejack",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Amontillado Sherry",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Pear Brandy",
+      unit: "oz",
+      unitAmount: 0.25,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Maple Syrup Grade A",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Apple Bitters",
+      unit: "Dash",
+      unitAmount: 3.0,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [],
+  portionlessIngredients: [],
+  garnishIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Allspice",
+      suffix: "Tincture",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Dehydrated",
+      ingredient: "Pear",
+      suffix: "Crisp",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Maple",
+      suffix: "Leaf",
```

Comment:
I'm 99% confident the ingredient name here should be "Maple Leaf". You can add an alias for "Maple" -> "Maple Leaf" if you want this recipe to show up when someone types in "Maple".

Reading this almost makes me want to remove the prefixes & suffixes entirely. I know their usage is a little subjective, but the whole ingredient name should usually go in the `ingredient` field; I was using it as a shortcut mostly for garnishes that had like "Lemon Wheel", "Grated Nutmeg", etc. The point of it was to keep the lists shorter for ingredients whose prep might take a second, like slicing the lemon or grating the nutmeg & reducing search time by keeping the ingredient (and so, also the alias) list shorter.

Reply:
My thinking (AFAIU) is that "Maple" is just a type of tree, whereas "Maple Leaf" is the actual ingredient you need.